src/main/java/seedu/address/commons/util/ModuleUtil.java

```diff
@@ -7,7 +7,7 @@
  */
 public class ModuleUtil {
```

Comment:
I just noticed this. Please move `ModuleUtil.java` from `seedu.address.commons.util` to `seedu.address.model.util`. `seedu.address.commons` should be treated as a common library that is applicable for any type of software, so nothing specific to Module Planner should live here.